chore(eslint): integrate eslint

obulaworld opened the pull request:
#### What does this PR do?
-Integrate ESlint to the project

#### Description of Task to be completed?
- Integration of eslint as a linting tool to make sure the right coding conventions are used.

#### How should this be manually tested?
- Deos not need a test

#### What are the relevant pivotal tracker stories?
[Configure ESLInt](https://www.pivotaltracker.com/story/show/160451588 "Configure ESLint")

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions

closes 160451588

Mrshinezee reviewed (approved):
LGTM

andela-phemmy commented:
LGTM

andela-phemmy reviewed (approved):
LGTM

KaiserPhemi commented on .eslintrc.js:
```diff
@@ -0,0 +1,33 @@
+          }
+      }]
+    }
+};
```
#### Nitpick
Insert new line at the end of file.

obulaworld replied:
Okay.On it now.
Thanks

obulaworld replied:
Added the new line to .eslintrc.js

Commit:
chore(eslint): integrate eslint

Integrate Eslint to the project
to make sure the team is following coding conventions (added new line)

closes 160451588
